.rubocop.yml

```diff
@@ -49,73 +49,49 @@ Rails/Exit:
   Exclude:
     - lib/decidim/git_backport_manager.rb
 
-# fix these rules later
-Capybara/SpecificMatcher:
-  Enabled: false
+RSpec/NoExpectationExample:
+  Exclude:
+    - decidim-admin/spec/system/participatory_space_private_user_spec.rb
```

Comment:
I think we should eventually fix these issues but is the plan to do it in a separate PR?

I checked for example this spec and it does not make any sense without an expectation, it is the same as not having that spec at all.

The next one kind of makes some sense but not much without an expectation.

On the other hand, I totally understand that fixing all these requires some work, so that's why I'm asking if the plan is to solve these in a separate PR.

Reply:
OK, I would suggest that we add these to the original issue to be fixed "later".

This is quite many specs which may not make much sense unless we fix them. But totally understand we may not be able to fix them straight away due to redesign.